src/components/notifications/SMSPartner.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+    <div class="mb-3">
+        <label for="smspartner-key" class="form-label">{{ $t("smspartnerApikey") }}</label>
```

Comment:
For common translations (such as this one), we try to stick to the actual words as keys to deduplicate them => reducing the translation effort.

```suggestion
        <label for="smspartner-key" class="form-label">{{ $t("API Key") }}</label>
```